src/components/Alert/Alert.Playground.stories.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { Meta, Story } from '@storybook/react/types-6-0';
+import Alert, { AlertProps } from './Alert';
+
+const alertVariants = [
```

Comment:
Use the `AlertVariants` defined in Alert.tsx instead of creating the same thing here.